Documentation/component-description.md

```diff
@@ -59,30 +62,51 @@ In this page, we focus on the status of each elements. For requirements and desi
     - More items are planned.
 - [tensor\_mux](../gst/nnstreamer/tensor_mux) (stable)
 - [tensor\_demux](../gst/nnstreamer/tensor_demux) (stable)
-- [tensor\_source](../gst/nnstreamer/tensor_source) (planned)
+- [tensor\_source](../gst/nnstreamer/tensor_source) (stable for IIO. More sources coming soon)
 - [tensor\_save](../gst/nnstreamer/tensor_saveload) (planned)
 - [tensor\_load](../gst/nnstreamer/tensor_saveload) (planned)
 - [tensor\_aggregator](../gst/nnstreamer/tensor_aggregator) (stable)
-- [tensor\_ros\_sink](https://github.com/nnsuite/nnstreamer-ros) (planned)
-- [tensor\_ros\_src](https://github.com/nnsuite/nnstreamer-ros) (planned)
+- [tensor\_ros\_sink](https://github.com/nnsuite/nnstreamer-ros) (stable for ROS1)
+- [tensor\_ros\_src](https://github.com/nnsuite/nnstreamer-ros) (stable for ROS1)
+- [tensor\_repo\_sink](../gst/nnstreamer/tensor_repo) (stable)
+- [tensor\_repo\_src](../gst/nnstreamer/tensor_repo) (stable)
 
 
 Note that test elements in /tests/ are not elements for applications. They exist as scaffoldings to test the above elements especially in the case where related elements are not yet implemented.
 
+# API Support
+
+- C-API
+  - Main target is Tizen, but supports other OS as well.
+  - [Implementation](../api/capi) (stable, since Tizen 5.5 M2)
+- C#-API (.NET)
+  - Main target is Tizen, but supports other OS as well.
+  - [Implementation](https://github.com/Samsung/TizenFX/pull/940) (to be released for Tizen 5.5 M2)
+- JAVA-API (Android)
+  - [Android sample apps in nnstreamer-example.git](https://github.com/nnsuite/nnstreamer-example/tree/master/android/example_app) uses JAVA APIs to implement Android-nnstreamer apps.
```

Comment:
how about linking the sample directory in this repo? actually all the android-application in nnstreamer-example does not use the APIs.

Reply:
As the sample apps in nnstreamer-example are not using JAVA API, yet, I'll update this